supplement/integration-testing/hibernate4-cache-over-mysql/src/test/java/com/github/shyiko/rook/it/h4com/IntegrationTest.java

```diff
@@ -52,14 +49,7 @@
 import java.sql.Connection;
 import java.sql.DriverManager;
 import java.sql.Statement;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.Map;
-import java.util.Properties;
-import java.util.ResourceBundle;
+import java.util.*;
```

Comment:
@igrunskiy have you tried running `mvn checkstyle:check`? I would expect this to fail because of AvoidStarImport rule.